src/ovh-ui-angular/ovh-ui-angular.routes.js

```diff
@@ -0,0 +1,56 @@
+import introductionTemplate from 'ovh-ui-angular/README.md'
+import checkboxTemplate from 'ovh-ui-angular/packages/oui-checkbox/README.md'
+import radioTemplate from 'ovh-ui-angular/packages/oui-radio/README.md'
+import radioGroupTemplate from 'ovh-ui-angular/packages/oui-radio-group/README.md'
+import messageTemplate from 'ovh-ui-angular/packages/oui-message/README.md'
+import contributingTemplate from 'ovh-ui-angular/CONTRIBUTING.md'
+
+import showCurrentValueInPopopController from './show-current-value-in-popup.controller'
+
+export default function ($stateProvider) {
+  'ngInject'
+
+  $stateProvider
+    .state('showcase.oui-angular', {
+      url: '/oui-angular',
+      friendlyName: 'oui-angular',
+      groupName: 'oui-angular components',
+      redirectTo: 'showcase.oui-angular.introduction',
+      template: '<ui-view></ui-view>',
+      weight: 9000
+    })
+    .state('showcase.oui-angular.introduction', {
+      url: '/introduction',
+      friendlyName: 'Introduction',
+      template: introductionTemplate
+    })
+    .state('showcase.oui-angular.contributing', {
+      url: '/contributing',
+      friendlyName: 'Contributing',
+      template: contributingTemplate
+    })
+    .state('showcase.oui-angular.checkbox', {
```

Comment:
We should do the same thing that we are doing with ovh-ui-kit components where we automaticaly discover components and import them.

Reply:
It's not as simple as for ui-kit I think